pkg/vulndash/adapter/adapter.go

```diff
@@ -103,7 +110,7 @@ func GetAllVulnerabilities(
 		var err error
 		occ, err = it.Next()
 		if err == iterator.Done {
-			logrus.Debug("no more occurrencess")
+			logrus.Infof("no more occurrences")
```

Comment:
I think we could keep this as debug (or remove altogether) since we will always print `done listing the vulnerabilities` right after this

Reply:
done